alf/utils/plot_tb_curves.py

```diff
@@ -393,27 +432,40 @@ def _clip_y(y):
             return np.clip(y, y_clipping[0],
                            y_clipping[1]) if y_clipping else y
 
+        if not isinstance(linestyle, list):
+            linestyle = [linestyle] * len(mean_curves)
```

Comment:
Why need to to this conversion to the style string? Is there an usecase where user can specify a list of style strings that has the same length as the curve?

Reply:
> Why need to do this conversion to the style string? Is there a use case where user can specify a list of style strings that has the same length as the curve?

If you want to mix dashed line styles with solid lines styles for examples (See my paper's curves)

Reply:
This needs to be documented somewhere. Currently the docstr ``linestyle (str)`` is not updated.

Reply:
updated